File: Pemrograman-1/Modul 6/C/PRAK603-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);
    if (n1 != n2) {
        printf("Jumlah tidak sama");
        return 0;  
    }
    int set1[n1];
    for (int i = 0; i < n1; i++) {
        scanf("%d", &set1[i]);
    }
    int set2[n2];
    for (int i = 0; i < n2; i++) {
        scanf("%d", &set2[i]);
    }
    for (int i = 0; i < n1; i++) {
        printf("%d ", set1[i] * set2[i]);
    }
    printf("\n");

    return 0;  
}

File: Semester-1/Pemrograman-1/Modul-6/Python/PRAK603-2310817210014-DamarjatiSuryoLaksono.py
n1, n2 = map(int, input().split())

if n1 != n2:
    print("Jumlah tidak sama")
    exit()

set1 = list(map(int, input().split()))
set2 = list(map(int, input().split()))

for i in range(n1):
    print(set1[i] * set2[i], end=" ")

print()
